forge-game/src/main/java/forge/game/ability/effects/FlipCoinEffect.java

```diff
@@ -196,57 +130,59 @@ public void resolve(SpellAbility sa) {
                         host.addRemembered(tempRemembered);
                     }
                 }
-            } else if (victory) {
-                if (sa.hasParam("RememberWinner")) {
-                    host.addRemembered(flipper);
-                }
-
-                if (sa.hasAdditionalAbility("WinSubAbility")) {
-                    AbilityUtils.resolve(sa.getAdditionalAbility("WinSubAbility"));
-                }
             } else {
-                if (sa.hasParam("RememberLoser")) {
-                    host.addRemembered(flipper);
+                int countWins = flipCoins(flipper, sa, amount);
+                int countLosses = Math.abs(countWins - amount);
+                if (countWins > 0) {
+                    if (sa.hasParam("RememberWinner")) {
+                        host.addRemembered(flipper);
+                    }
+                    SpellAbility sub = sa.getAdditionalAbility("WinSubAbility");
+                    if (sub != null) {
+                        sub.setSVar("Wins", "Number$" + countWins);
+                        AbilityUtils.resolve(sub);
+                    }
                 }
-
-                if (sa.hasAdditionalAbility("LoseSubAbility")) {
-                    AbilityUtils.resolve(sa.getAdditionalAbility("LoseSubAbility"));
+                if (countLosses > 0) {
+                    if (sa.hasParam("RememberLoser")) {
+                        host.addRemembered(flipper);
+                    }
+                    SpellAbility sub = sa.getAdditionalAbility("LoseSubAbility");
+                    if (sub != null) {
+                        sub.setSVar("Losses", "Number$" + countLosses);
+                        AbilityUtils.resolve(sub);
+                    }
+                }
+                if (sa.hasParam("RememberNumber")) {
+                    String toRemember = sa.getParam("RememberNumber");
+                    if (toRemember.startsWith("Win")) {
+                        host.addRemembered(countWins);
+                    } else if (toRemember.startsWith("Loss")) {
+                        host.addRemembered(countLosses);
+                    }
                 }
             }
         }
     }
 
-    /**
-     * <p>
-     * flipCoinNoCall  Flip a coin without any call.
-     * </p>
-     *
-     * @param sa   the source card.
-     * @param flipper  the player flipping the coin.
-     * @param multiplier
-     * @return a boolean.
-     */
-    public boolean flipCoinNoCall(final SpellAbility sa, final Player flipper, final int multiplier, final String varName) {
-        boolean result = false;
-        int numSuccesses = 0;
-
+    public static int flipCoins(final Player flipper, final SpellAbility sa, final int amount) {
+        return flipCoins(flipper, sa, amount, "");
+    }
+    public static int flipCoins(final Player flipper, final SpellAbility sa, final int amount, final String info) {
+        int multiplier = getFlipMultiplier(flipper);
```

Comment:
@tool4ever i wish we could do something about this hidden Keyword: 
`"If you would flip a coin, instead flip two coins and ignore one."`